Tasks/PackerBuildV0/src/utilities.ts

```diff
@@ -84,6 +84,17 @@ export function readJsonFile(filePath: string): string {
     return content;
 }
 
+export function createTemplateVarFile (templateVariables: Map<string, string>): string {
+    let filePath: string = path.resolve(tl.getVariable('Agent.TempDirectory'), Math.random().toString(36).replace('0.', '') + '.json');
+    let res = {};
+    templateVariables.forEach((value: string, key: string) => {
+        res[key] = value
+    });
+    let content: string = JSON.stringify(res);
```

Comment:
Please test with special characters in values - space, quotes etc

Reply:
Tested this on ubuntu 18.04 and win 10. The var file created had whitespaces and quotes in its path.

Reply:
You should also test for variable values containing special chars. They should be properly escaped when using Json.Stringify and packer should be able to consume them successfully

Reply:
done.